x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/flyout_button.cy.ts

```diff
@@ -64,18 +63,4 @@ describe.skip('timeline flyout button', () => {
       cy.get(TIMELINE_BOTTOM_BAR_TOGGLE_BUTTON).should('have.focus');
     }
   );
-
-  it(
```

Comment:
Could you explain that happened to this test?

Reply:
I thought this test was completely irrelevant to flyout_button and was also not sure what was it actually testing. What we need to test global dropdown here?